Darwin/Services/StitchRemoteMongoDBService/StitchSyncPerformanceTests/SyncPerformanceTestUtils.swift

```diff
@@ -0,0 +1,136 @@
+
+    static let configuredStitchHostName: String = {
+        if !TEST_PERF_IOS_STITCH_HOST.isEmpty {
+            if TEST_PERF_IOS_STITCH_HOST.lowercased() == "local" {
+                return defaultStitchHostName
+            } else if TEST_PERF_IOS_STITCH_HOST.lowercased() == "prod" {
+                return "https://stitch.mongodb.com"
+            }
+        } else if let param = ProcessInfo.processInfo.environment["PERF_IOS_STITCH_HOST"] {
+            return param
+        }
+        return defaultStitchHostName
+    }()
+
+    static let configuredNumIters: Int = {
+        if let param = Int(TEST_PERF_IOS_NUM_ITERS) {
+            return param
+        } else if let paramStr = ProcessInfo.processInfo.environment["PERF_IOS_NUM_ITERS"], let param = Int(paramStr) {
+            return param
+        }
+        return defaultNumIters
+    }()
+
+    static let configuredHostname: String = {
+        if !TEST_PERF_IOS_HOSTNAME.isEmpty {
+            return TEST_PERF_IOS_HOSTNAME
+        } else if let param = ProcessInfo.processInfo.environment["PERF_IOS_HOSTNAME"] {
+            return param
+        }
+        return defaultHostName
+    }()
+
+    static let configuredDocSizes: [Int] = {
+        if !TEST_PERF_IOS_DOC_SIZES.isEmpty {
+            return stringToIntArr(TEST_PERF_IOS_DOC_SIZES)
+        } else if let param = ProcessInfo.processInfo.environment["PERF_IOS_DOC_SIZES"] {
+            return stringToIntArr(param)
+        }
+        return defaultDocSizes
+    }()
+
+    static let configuredNumDocs: [Int] = {
+        if !TEST_PERF_IOS_NUM_DOCS.isEmpty {
+            return stringToIntArr(TEST_PERF_IOS_NUM_DOCS)
+        } else if let param = ProcessInfo.processInfo.environment["PERF_IOS_NUM_DOCS"] {
+            return stringToIntArr(param)
+        }
+        return defaultNumDocs
+    }()
+
+    static let configuredDataGranularity: Double = {
+        if let param = Double(TEST_PERF_IOS_DATA_GRANULARITY) {
+            return param
+        } else if let paramStr = ProcessInfo.processInfo.environment["PERF_IOS_DATA_GRANULARITY"],
+            let param = Double(paramStr) {
+            return param
+        }
+        return defaultDataGranularity
+    }()
+
+    static let configuredNumOutliers: Int = {
+        if let param = Int(TEST_PERF_IOS_NUM_OUTLIERS) {
+            return param
+        } else if let paramStr = ProcessInfo.processInfo.environment["PERF_IOS_NUM_OUTLIERS"],
+            let param = Int(paramStr) {
+            return param
+        }
+        return defaultNumOutliers
+    }()
+
+    static let configuredShouldOutputToStdOut: Bool = {
+        if let param = Bool(TEST_PERF_IOS_OUTPUT_STDOUT) {
+            return param
+        } else if let paramStr = ProcessInfo.processInfo.environment["PERF_IOS_OUTPUT_STDOUT"],
+            let param = Bool(paramStr) {
+            return param
+        }
+        return defaultOutputToStdout
+    }()
+
+    static let configuredShouldOutputToStitch: Bool = {
```

Comment:
[q] Why are all of these fields preceded by `configured`? Seems redundant.

Reply:
Just a pattern I took from the kotlin code